prometheus/promhttp/http.go

```diff
@@ -54,6 +55,26 @@ const (
 	processStartTimeHeader = "Process-Start-Time-Unix"
 )
 
+type Compression int
+
+const (
+	Identity Compression = iota
+	Gzip
+	Zstd
+)
+
+var compressions = [...]string{
+	"identity",
+	"gzip",
+	"zstd",
+}
+
+func (c Compression) String() string {
+	return compressions[c]
+}
```

Comment:
If we use `string` instead of `iota` we don't need this code, right? (:

Reply:
Yes, you're right! Thanks!